mesonbuild/environment.py

```diff
@@ -1016,6 +1016,37 @@ def detect_compilers(self, lang, need_cross_compiler):
                 raise EnvironmentException('Cross compilation with Swift is not working yet.')
                 # cross_comp = self.environment.detect_fortran_compiler(True)
 
+        if comp is None:
```

Comment:
...if you make this a final `else` and the rest `elif`s

Reply:
Obsolete with the last commit.